packages/aws-cdk-lib/aws-ec2/lib/nat.ts

```diff
@@ -188,6 +188,13 @@ export interface NatInstanceProps {
    */
   readonly instanceType: InstanceType;
 
+  /**
+   * Whether to associate a public IP address to the primary network interface attached to this instance.
+   *
+   * @default false
```

Comment:
I believe here you are referring to the CFN default value, but not the CDK one. I think it is better to mention that default value is null.